src/modules/poll.command.js

```diff
@@ -0,0 +1,30 @@
+
+const command = require('./command');
+
+/**
+ * poll() implements the polls logic
+ * input line for creating a poll should look like /poll <question sentence>? <answer1> <"answer 2"> ...
+ * quotes are used to ignore spaces inside the answers
+ */
+const poll = async function(response, { input, i18n }) {
+    poll_obj = {};
+
+    // separating the question by question mark
+    poll_obj.question = input.substring(input.indexOf(' ') + 1 , input.indexOf('?'));
```

Comment:
Semi-offtop: it restrict us to poll form, btw. I mean, we always forced to formulate poll with question mark.
Also, look for `response.args` inside `parseCommand` module, maybe we need to update it to fit this case (with quotes). And it is solving the issue with `?`. If we don't mind this format, of course:
`/poll "some question/suggestion here" variant1 "variant2 with spaces"`